framework/mesh/MeshGenerator/OrthogonalMeshGenerator.cc

```diff
@@ -14,7 +14,7 @@ chi::InputParameters OrthogonalMeshGenerator::GetInputParameters()
   chi::InputParameters params = MeshGenerator::GetInputParameters();
 
   params.SetGeneralDescription("Creates orthogonal meshes.");
-  params.SetDocGroup("MeshGenerator");
+  params.SetDocGroup("doc_MeshGenerators");
```

Comment:
same Q about the param doc. maybe I do not understand what it should be. is doc_MeshGenerators ?

Reply:
Tag to the main page for all MeshGenerators.